modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties

```diff
@@ -2,11 +2,15 @@ add-new-related-queries-based-on-successful-queries=Add New Related Queries Base
 add-x-as-a-search-provider=Add {0} as a Search Provider
 aggregation-field=Aggregation Field
 aggregation-field-help=The name of the indexed field to aggregate results by. Must be a non-analyzed keyword field.
+allow-empty-searches=Allow Empty Searches
+allow-empty-searches-help=When enabled, an empty search query returns all available assets. By default, an empty query returns no search results.
```

Comment:
@arboliveira I already reviewed this one when @brandizzi sent it to me, so it looks good.